src/client/TableRow.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class TableRow extends React.Component{
+  render(){
+    const isEven = (this.props.seq % 2 === 0) ? true : false;
```

Comment:
With `ternary operator`, we'll use the value between `?` and `:` for truthy condition and the value after `:` for falsy condition.

So, if we'd like to just get `Boolean` result, we might not need `ternary operator` here.
Maybe using the condition to get the `Boolean` result is enough?